solutions/swb-app/src/validatorHelper.ts

```diff
@@ -22,22 +21,4 @@ function processValidatorResult(validatorResult: ValidatorResult): void {
   }
 }
 
-function validateAndParse<T>(parser: ZodTypeAny, data: unknown): T {
```

Comment:
Wait is this supposed to be removed? I think other functions use this method.

Reply:
It was moved so downstream packages can use this method: https://github.com/aws-solutions/solution-spark-on-aws/pull/641/files#diff-5619b93169c6639e8622293c9d180c4ed17a51fa2f9e0eda913aa3fcdf9f7fdd